gradle/javadoc.gradle

```diff
@@ -0,0 +1,19 @@
+apply from: "${rootDir}/gradle/dependencies.gradle"
+
+android.libraryVariants.all { variant ->
```

Comment:
This file means that `/library/javadoc.gradle` can be removed.

Reply:
Removed